README.md

```diff
@@ -10,3 +10,30 @@ It analyses the QGIS project and suggests and performs actions needed to make th
 More information can be found in the [QField documentation](https://docs.qfield.org/get-started/).
 
 The plugin can be download on the [QGIS plugin repository](https://plugins.qgis.org/plugins/qfieldsync/).
+
+## Development
+
+1) Checkout [qfieldsync](https://github.com/opengisch/qfieldsync/) locally:
+
+```
+git clone --recurse-submodules git@github.com:opengisch/qfieldsync.git
+```
+
+2) Make a link of the QFieldSync checkout to qfieldsync directory in your current QGIS profile:
+
+```
+ln -s ${PWD}/qfieldsync/qfieldsync ${HOME}/.local/share/QGIS/QGIS3/profiles/default/python/plugins
+```
+
+3) Checkout [libqfieldsync](https://github.com/opengisch/libqfieldsync/) locally:
+
+```
+git clone git@github.com:opengisch/libqfieldsync.git
+```
+
+4) Install your local libqfieldsync as editable dependency (assuming you are in the same directory as step 3):
+
+```
+pip install -e libqfieldsync
```

Comment:
on ubuntu 23.10 (and on latest debian too I think), you'll need to add a scary --break-system-packages to that line.

Reply:
Accomodated.